packages/process/_pipe.pony

```diff
@@ -172,14 +178,17 @@ class _Pipe
           bytes_to_read, addressof bytes_read, USize(0))
       let winerr = @GetLastError[I32]()
       if not ok then
-        if (winerr == _ERRORBROKENPIPE()) or (winerr == _ERRORNODATA()) then
```

Comment:
_ERRORNODATA should be impossible to get here based on doing non blocking and us having previously peeked to know there is data to read.